Space-Acres.app/Contents/Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>CFBundleExecutable</key>
+    <string>space-acres</string>
+    <key>CFBundleIdentifier</key>
+    <string>com.subspace.spaceacres</string>
+    <key>CFBundleName</key>
+    <string>Space Acres</string>
+    <key>CFBundleVersion</key>
+    <string>0.1.17</string>
+    <key>CFBundleShortVersionString</key>
+    <string>0.1.17</string>
+    <key>CFBundlePackageType</key>
+    <string>APPL</string>
+    <key>CFBundleSignature</key>
+    <string>????</string>
+    <key>CFBundleInfoDictionaryVersion</key>
+    <string>6.0</string>
+    <key>LSMinimumSystemVersion</key>
+    <string>10.12</string>
```

Comment:
What do these versions correspond to?

Reply:
`CFBundleInfoDictionaryVersion` is normally [assigned by Xcode](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleinfodictionaryversion). Since this application wasn't built with Xcode, I assigned it manually. `6.0` is a common value and it should be compatible with most MacOS devices.

`LSMinimumSystemVersion` is the [lowest MacOS version](https://developer.apple.com/documentation/bundleresources/information_property_list/lsminimumsystemversion) that the program can run on. I specified Sierra. Needs testing/verification.